src/api/services/auth.service.js

```diff
@@ -8,3 +8,15 @@ exports.checkUser = async email => {
 exports.saveToken = async (email, token) => {
   await User.findOneAndUpdate({ email }, { token }).exec();
 };
+
+exports.logout = async user => {
+  try {
+    const { _id } = user;
+
+    await User.findByIdAndUpdate(_id, {
+      token: null,
+    }).exec();
```

Comment:
await가 있으면 exec가 필요 없지 않나 싶어, exec가 꼭 필요한가에 대해서 좀 찾아보았습니다. 기능적인 측면에서는 동일하지만 mongoose 문서에서는 exec를 권장하고 있기는 하네요. 👍

[Should You Use exec() With await?](https://mongoosejs.com/docs/promises.html#should-you-use-exec-with-await)
> There are two alternatives for using await with queries:
> 
> await Band.findOne();
> await Band.findOne().exec();
> 
> As far as functionality is concerned, these two are equivalent. However, we recommend using .exec() because that gives you better stack traces.